src/Playwright.Tests/BrowserTypeConnectTests.cs

```diff
@@ -91,7 +98,6 @@ public void ShouldRejectNavigationWhenBrowserCloses()
         }
 
         [PlaywrightTest("browsertype-connect.spec.ts", "should reject waitForSelector when browser closes")]
-        [Test, Ignore("SKIP WIRE")]
```

Comment:
Test that's empty should not have `Ignore` removed. Either implement the test, or leave the `Ignore`.

Reply:
Added back